lib/node_modules/@stdlib/array/base/count-same-value/lib/main.js

```diff
@@ -132,6 +135,46 @@ function complex( x, value ) {
 	return n;
 }
 
+/**
+* Counts the number of elements in a boolean array that are equal to a specified value.
+*
+* @private
+* @param {Collection} x - input array
+* @param {*} value - search value
+* @returns {NonNegativeInteger} number of elements that are equal to a specified value
+*
+* @example
+* var BooleanArray = require( '@stdlib/array/bool' );
+*
+* var x = new BooleanArray( [ true, false, true, false, true ] );
+*
+* var n = boolean( x, true );
+* // returns 3
+*/
+function boolean( x, value ) {
+	var view;
+	var n;
+	var i;
+
+	if ( !isBoolean( value ) ) {
+		return 0;
+	}
+	if ( value === false ) {
+		value = 0;
+	} else {
+		value = 1;
+	}
+	view = reinterpretBoolean( x, 0 );
+
+	n = 0;
+	for ( i = 0; i < view.length; i++ ) {
+		if ( isSameValue( view[ i ], value ) ) {
```

Comment:
```suggestion
		if ( view[ i ] === value ) {
```

We can avoid the function call entirely here, as we are comparing integers.